javascript/ql/src/experimental/poi/PoI.qll

```diff
@@ -0,0 +1,329 @@
+/**
+ * Provides classes and predicates for discovering points of interest
+ * in an unknown code base.
+ *
+ * To use this module, subclass the
+ * `PoI` class, override *one* of its `is` predicates, and use
```

Comment:
```suggestion
 * To use this module, subclass the
 * `ActivePoi` class, override *one* of its `is` predicates, and use
```

Shouldn't only standard library PoI classes extends `PoI` directly, the rest should extend `ActivePoI`, right?

Reply:
Right. If you start extending `PoI` by yourself, then you must also use a `PoIConfiguration` to make use of it, so this documentation piece should just mention `ActivePoI`.